chapters/CRUD-Operations.md

```diff
@@ -32,19 +41,26 @@ corresponding initial DID document.
 1. Set `initialDocument` to the result of passing `did` into the [Read] algorithm.
 1. Return `did` and `initialDocument`.
 
-#### External Initial Document Creation
+#### From External Intermediate DID Document
+
+The From External ::Intermediate DID Document:: Creation algorithm enables the ability 
+to create a **did:btc1** from some intermediate DID document. This allows for a
+more complex ::Initial DID Document::, including the ability to include
+Service Endpoints and ::Beacons:: that support aggregation.
 
-It is possible to create a **did:btc1** from some initiating arbitrary DID document.
-This allows for more complex initial DID documents, including the ability to include
-service endpoints and ::Beacons:: that support aggregation.
+It takes the following inputs:
+* `intermediateDocument`: any arbitrary, valid DID document with all `identifier`
+   fields (e.g., the id field) replaced with the placeholder value
```

Comment:
```suggestion
* `intermediateDocument`: any arbitrary, valid DID document with the `identifier`
   replaced with the placeholder value throughout all fields (e.g. the id field)
```

Reply:
I think this is clearer. Because there are times where e.g. vm.id = <identifier>#key-1. Which should go to <placeholder>#key-1